SashaZaitsev/HW5/05. Loops.py

```diff
@@ -0,0 +1,26 @@
+# Программа, що перетворює список з класу int на float
+numbers = [1, 2 , 3, 4, 5]
+print("Перетворюємо числа у тип float")
+
```

Comment:
look at this implementation
list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)